index.html

```diff
@@ -671,17 +671,28 @@
     <div class="modal-dialog modal-lg">
       <div class="modal-content">
         <div class="well">
-          <h3>Create Bash Script named “Rewrap-MXF.sh” to do Batch FFmpeg Processing</h3>
-          <p><code>for f in *.MXF; do ffmpeg -i "$f" -map 0 -c copy "${f%.MXF}.mov"; done</code></p>
-          <p>Re-wrap <code>.MFX</code> files in a specified directory to <code>.mov</code> files by using this code within a <code>.sh</code> file. The shell script (.sh file) and all MXF files must be contained in the same directory, and the script must be run from the directory itself (cd ~/Desktop/MXF_file_directory). Execute <code>.sh</code> file with the command <code>sh Rewrap-MXF.sh</code></p>
+          <h3>Create Bash script to batch process with ffmpeg</h3>
```

Comment:
I guess FFmpeg is better in this case, because the principle is also very useful for `ffprobe`, not «only» for `ffmpeg`.